packages/pigeon/mock_handler_tester/test/message.dart

```diff
@@ -0,0 +1,147 @@
+// Autogenerated from Pigeon (v0.1.0), do not edit directly.
```

Comment:
This file gets regenerated when running  `run_tests.sh`.  I still don't have the time to invest in creating a mechanism to generate these during standard execution of the tests.